cdk/lib/facia-purger.ts

```diff
@@ -1,15 +1,83 @@
-import { join } from 'path';
 import type { GuStackProps } from '@guardian/cdk/lib/constructs/core';
 import { GuStack } from '@guardian/cdk/lib/constructs/core';
+import { GuLambdaFunction } from '@guardian/cdk/lib/constructs/lambda';
 import type { App } from 'aws-cdk-lib';
-import { CfnInclude } from 'aws-cdk-lib/cloudformation-include';
-
+import { CfnParameter, Duration } from 'aws-cdk-lib';
+import {
+	Alarm,
+	ComparisonOperator,
+	TreatMissingData,
+	Unit,
+} from 'aws-cdk-lib/aws-cloudwatch';
+import { SnsAction } from 'aws-cdk-lib/aws-cloudwatch-actions';
+import { Runtime } from 'aws-cdk-lib/aws-lambda';
+import { Topic } from 'aws-cdk-lib/aws-sns';
+import { EmailSubscription } from 'aws-cdk-lib/aws-sns-subscriptions';
 
 export class FaciaPurger extends GuStack {
 	constructor(scope: App, id: string, props: GuStackProps) {
 		super(scope, id, props);
 
-		new CfnInclude(this, "newFaciaPurgerId", {templateFile:"../facia-purger/cloudformation.yaml"} )
+		//new CfnInclude(this, "newFaciaPurgerId", {templateFile:"../facia-purger/cloudformation.yaml"} )
+
+		const fastlyServiceId = new CfnParameter(this, 'FastlyServiceId', {
+			type: 'String',
+			description: 'Id of service to purge',
+		});
+		const fastlyAPIKey = new CfnParameter(this, 'FastlyAPIKey', {
+			type: 'String',
+			description:
+				'API key with purge writes for service with id FastlyServiceId',
+			noEcho: true,
+		});
+		const emailRecipient = new CfnParameter(this, 'EmailRecipient', {
+			type: 'String',
+			description: 'The email address to send alarm notifications to',
+		});
```

Comment:
I'd recommend making this a normal variable, rather than a CFN Parameter, as the value is not private or secret.

Reply:
Interesting… we would have to pull this value down, as it was not currently defined in code. We can refactor in a subsequent PR.